Preface/Preface.pillar

```diff
@@ -58,12 +57,13 @@ available at *http://www.deepintopharo.org*
 
 Do not be frustrated by parts of Pharo that you do not immediately understand.
 You do not have to know everything! Alan Knight expresses this as follows:
-
+{{{latex:\begin{quote} }}}
```

Comment:
@dmitrizagidulin When you prefix a line with `@@XXX` Pillar generates a LaTeX environment named `XXX`. It's up to the book author to define this `XXX` LaTeX environment. This is typically in a file named `support/latex/common.tex`.

Use `@@quote` when it's a quote. `@@note` should generate a paragraph with "Note" and not "Notes". Please fix it.